prisma/seed.ts

```diff
@@ -0,0 +1,52 @@
+import { PrismaClient } from '@prisma/client'
+
+
+//// Instantiate Prisma Client
+const prisma = new PrismaClient()
+
+async function main(){
+    //just for testing purpose
+    await prisma.user.deleteMany({})
+
+
+    const user= await prisma.user.create({
+        data:{
+            id: '1',
+            email: 'alice@gmail.com',
+            timeZone: 'Chicago/USA',
+            availability: {
+                create: [
+                {
+                    id:'11',
+                    startTime:'new Date(2021, 0O5, 0O5, 17, 00, 00, 00)',
```

Comment:
Consider `new Date().toIsoString()`

Reply:
new Date().toIsoString() will yield string instead of DateTime?

Reply:
My mistake, I would just use `new Date()` without the quotations around it.